### DISCUSSION TITLE
npx pod-install says "CocoaPods is not supported for this project"

### DISCUSSION THREAD

**Discussion by wfettich** [dooboolab/react-native-audio-recorder-player#407] [Q&A] [answered]
After downloading the project and running npx pod-install , I get 
"CocoaPods is not supported for this project"

**Comment by hyochan** [comment]
Could you kindly test past version and recent version have same errors?

**Reply by hyochan** [reply]
Oh sorry I think I miss understood your question. Let me check example proj tommorow

**Comment by hyochan** [comment, marked as answer]
@wfettich I think you've not navigate into `Example` dir.

<img width="476" alt="Screen Shot 2021-12-05 at 12 42 18 PM" src="https://user-images.githubusercontent.com/27461460/144732540-dce0dc19-42c9-4a4d-bba1-6a7e92552a07.png">

I've just tried this in `master` branch.
